uploaders/aabuploader.go

```diff
@@ -68,6 +69,12 @@ func DeployAAB(item deployment.DeployableItem, artifacts []string, buildURL, tok
 		"build_type":      info.BuildType,
 	}
 
+	signature, err := androidsignature.Read(pth)
+	if err != nil {
+		log.Errorf("Failed to read signature: %s", err)
```

Comment:
This is more like a warning because this won't fail the step.

Reply:
I copied this from a few lines below so I fixed it at both places.